apps/tasks/ejs.js

```diff
@@ -2,6 +2,9 @@ module.exports = function(grunt) {
   'use strict';
 
   var ejs = require('ejs');
+  // Don't let ejs overload how the ejs extension is loaded, as this confuses
+  // browserify
+  require.extensions['.ejs'] = undefined;
```

Comment:
This is rather hacky, but I think ok as a transitional step.

What's happening here is that the ejs module has a custom handler for .ejs files. In this change I've made it so that when we translate the ejs file into a js file, it retains its ejs extension. This means we now have files with the name .ejs file which we want to be treated like regular js files, so I get rid of the .ejs handler.